packages/ifpack2/src/Ifpack2_IlukGraph.hpp

```diff
@@ -305,8 +305,13 @@ void IlukGraph<GraphType, KKHandleType>::initialize()
                            // Heuristic to get the maximum number of entries per row.
                            int RowMaxNumIndices = localOverlapGraph.rowConst(i).length;
                            numEntPerRow_d(i) = (levelfill == 0) ? RowMaxNumIndices  // No additional storage needed
-                             : Kokkos::Experimental::ceil(static_cast<double>(RowMaxNumIndices) 
+#ifdef KOKKOS_ENABLE_DEPRECATED_CODE_3
+                             : Kokkos::Experimental::ceil(static_cast<double>(RowMaxNumIndices)
                                     * Kokkos::Experimental::pow(overalloc, levelfill));
+#else
+                             : Kokkos::ceil(static_cast<double>(RowMaxNumIndices)
+                                    * Kokkos::pow(overalloc, levelfill));
+#endif
```

Comment:
Why would you keep the branch when deprecated code is enabled?
Just use the math functions from the Kokkos:: namespace from now on

Reply:
I can drop this, was being overly cautious at the time

Reply:
Please drop it

Reply:
Addressed by 54140b1f1d5170fac3de90ca36298c69dbb5b01e